src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -353,12 +365,12 @@ private void selectConversation(MouseEvent e) {
     * @param index  index of selected conversation in the list of conversations
     */
     private ConversationSummary lookupByTitle(String title, int index) {
-        int localIndex = 0;
         for (final ConversationSummary cs : clientContext.conversation.getConversationSummaries()) {
-            if ((localIndex >= index) && cs.title.equals(title)) {
+        	// An exception was thrown when localIndex was used and tested in the following if statement.
```

Comment:
Same comment as above. What if there were a million conversations? Aren't these stored in a database? Could you look them up that way?